src/test/regress/sql/with_clause.sql

```diff
@@ -1,3 +1,10 @@
+-- start_matchsubs
+--
+-- m/ERROR:  Multiple inlining of non-SELECT operations is prohibited \(allpaths\.c:\d+\)/
+-- s/\d+/XXX/g
+--
+-- end_matchsubs
```

Comment:
Can this matchsubs block can be moved closer to the new test cases?

Reply:
I think current variant can be treated as normal practice. So, it's quite unnecessary